package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "storj",
-  "version": "3.1.2",
+  "version": "3.1.3",
```

Comment:
3.2.0 (minor version increment - *added* functionality in a backwards compatible way)